crates/bevy_state/src/app.rs

```diff
@@ -55,6 +58,25 @@ pub trait AppExtStates {
     ///
     /// For more information refer to [`StateScoped`](crate::state_scoped::StateScoped).
     fn enable_state_scoped_entities<S: States>(&mut self) -> &mut Self;
+
+    #[cfg(feature = "bevy_reflect")]
+    /// Registers the state type `T` using `[App::register]`,
+    /// and adds [`crate::reflect::ReflectState`] type data to `T` in the type registry.
```

Comment:
Nit: I think we normally shorten these doc links. Something like:
```rust
    /// and adds [`ReflectState`] type data to `T` in the type registry.
    ///
    /// ...
    ///
    /// [`ReflectState`]: crate::reflect::ReflectState
```

Reply:
```suggestion
    /// and adds [`ReflectState`](crate::reflect::ReflectState) type data to `T` in the type registry.
```